manifests/t/Telegram/TelegramDesktop/4.14.3/Telegram.TelegramDesktop.installer.yaml

```
@@ -0,0 +1,44 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Telegram.TelegramDesktop
+PackageVersion: 4.14.3
+Platform:
+- Windows.Desktop
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Protocols:
+- tg
+ReleaseDate: 2024-01-04
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: Telegram\Telegram.exe
+    PortableCommandAlias: telegram
+  InstallerUrl: https://github.com/telegramdesktop/tdesktop/releases/download/v4.14.3/tsetup-x64.4.14.3.exe
+  InstallerSha256: C12620FC472A723515F67C7EB5C634470CFAC0CF4E986CF9E76FF68358D52DFC
+- Architecture: x86
+  InstallerType: inno
+  Scope: user
+  InstallerUrl: https://github.com/telegramdesktop/tdesktop/releases/download/v4.14.3/tsetup.4.14.3.exe
+  InstallerSha256: 8D898989A90ACB59352A21F6519EC96404F5D9597E47F08EE0DF3B60A1E39F96
+- Architecture: x64
+  InstallerType: zip
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: Telegram\Telegram.exe
+    PortableCommandAlias: telegram
+  InstallerUrl: https://github.com/telegramdesktop/tdesktop/releases/download/v4.14.3/tportable-x64.4.14.3.zip
+  InstallerSha256: 01C0E1E707DF47818FB42744214FE6472BB60DB5063F7C25C3E72C51630474C6
+- Architecture: x64
+  InstallerType: zip
+  Scope: user
+  InstallerUrl: https://github.com/telegramdesktop/tdesktop/releases/download/v4.14.3/tportable.4.14.3.zip
+  InstallerSha256: 26688F330B5438D356A44D9F7D917CF0C5EACFDAB46494165360E702745BEF9B

```

I believe this one should be the x86 portable version.

```suggestion
- Architecture: x86
  InstallerType: zip
  NestedInstallerType: portable
  NestedInstallerFiles:
  - RelativeFilePath: Telegram\Telegram.exe
    PortableCommandAlias: telegram
  InstallerUrl: https://github.com/telegramdesktop/tdesktop/releases/download/v4.14.3/tportable.4.14.3.zip
  InstallerSha256: 26688F330B5438D356A44D9F7D917CF0C5EACFDAB46494165360E702745BEF9B
```